test/parallel/test-fs-assert-encoding-error.js

```diff
@@ -4,72 +4,72 @@ const assert = require('assert');
 const fs = require('fs');
 
 const options = 'test';
-const unknownEncodingMessage = /^Error: Unknown encoding: test$/;
+const expectedError = common.expectsError('ERR_INVALID_OPT_VALUE');
```

Comment:
Please also check for the type as this is not done anymore with the current check. Please change this in all the test files accordingly.